lib/Dialect/Calyx/Export/CalyxEmitter.cpp

```diff
@@ -1034,10 +1039,17 @@ void Emitter::emitLibraryFloatingPoint(Operation *op) {
   if (auto fpOp = dyn_cast<calyx::FloatingPointOpInterface>(op)) {
     opName = fpOp.getCalyxLibraryName();
   }
+
   indent() << getAttributes(op, /*atFormat=*/true) << cell.instanceName()
            << space() << equals() << space() << opName << LParen() << expWidth
-           << comma() << sigWidth << comma() << bitWidth << RParen()
-           << semicolonEndL();
+           << comma() << sigWidth << comma() << bitWidth;
+  if (opName == "fpToInt") {
```

Comment:
Nit: is there something less brittle than the opName, e.g., perhaps a `Type` that we can dyn_cast?